po/content.de.po

```diff
@@ -211,7 +211,7 @@ msgstr "Keine Crapware"
 #. type: Bullet: '- '
 #: content/_index.md
 msgid "~~No pesticides~~ Purely organic!"
-msgstr "~~Keine Pestizide~~ Rein biologisch!"
+msgstr "~~Keine Pestizide~~ Rein und organisch!"
```

Comment:
the translation of 'pure and organic' in content/_index.de.md has been changed to 'rein und nachhaltig' (pure and sustainable), which sounds better to me. It should be changed here as well.